src/test/java/acceptance/input_from_stream_of_barcodes/CanVideoRecorderPlayWithDiffScreenMetrics.java

```diff
@@ -0,0 +1,76 @@
+package acceptance.input_from_stream_of_barcodes;
+
+import com.google.zxing.BarcodeFormat;
+import org.junit.Test;
+import tdl.record.screen.image.input.ImageInput;
+import tdl.record.screen.image.input.InputFromStreamOfBarcodes;
+import tdl.record.screen.time.FakeTimeSource;
+import tdl.record.screen.time.TimeSource;
+import tdl.record.screen.video.VideoRecorder;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+
+import static junit.framework.TestCase.fail;
+
+public class CanVideoRecorderPlayWithDiffScreenMetrics {
```

Comment:
There is no need for this test. The unit level test is enough to cover this edge case.